src/main/kotlin/org/jitsi/videobridge/stats/config/StatsManagerBundleActivatorConfig.kt

```diff
@@ -147,22 +143,6 @@ class StatsManagerBundleActivatorConfig {
              * Because of that, we define these here to handle the above use cases, but
              * implement the property that's used differently (see above)
              */
-            @Suppress("unused")
```

Comment:
Now that there is only one remaining property here, we can use a 'normal' property (instead of these three unused ones and the 'meta' property above).  We can save that for another PR though.